### PR TITLE
Add EDW glossary entry

### PR DESCRIPTION
**Pull request by johnblust**
## Description & motivation
Adds EDW glossary entry to the analytics engineering glossary

### CODE REVIEW

**Comment by KiraFuruichi** [comment]
I made some substantial changes to the written content [here](https://docs.google.com/document/d/1R7JDoDuI_Z4VHVOtq49tX75qHAquKefU15Vu8h8HrIY/edit#), lmk if you have any questions or want to chat through this version I have!

**Comment by johnblust** [comment]
You killed it, Kira! I responded to your one comment, but I think I'm going to start adding this content to the PR

**Comment by johnblust** [comment]
@KiraFuruichi Okay, I added the new content. You can view the [preview here](https://deploy-preview-1846--docs-getdbt-com.netlify.app/terms/edw). If it looks good, go ahead and approve the PR so I can merge. Thanks!!

File: website/docs/terms/edw.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+In the world of analytics engineering, most teams have one central data warehouse that houses data from all of their different departments and functions.
+
+
+### Why is this distinction necessary? 
+
+One of the main distinctions is in an organization’s users and distribution. If an organization has multiple databases, a central data warehouse is used to create separate entities between raw and source data, staging work, and ready-for-use analytics datasets. In this EDW and classic data warehouse setup, data is accessible across an organization, data teams can create <Term id="table">tables</Term> that join data from multiple sources, and users can gain enriched perspectives into their data.
+If a company has very siloed departments that manage their own data, budgets, and have little need for crossover with other departments or data sources, emphasizing the difference between a central EDW data warehouse and their own data warehouse could be a necessity for budgeting and governance reasons.
```
Comment (KiraFuruichi) [inline website/docs/terms/edw.md:28]:
```suggestion

If a company has very siloed departments that manage their own data, budgets, and have little need for crossover with other departments or data sources, emphasizing the difference between a central EDW data warehouse and their own data warehouse could be a necessity for budgeting and governance reasons.
```

Add additional spacing to match paragraph spacing in draft

File: website/docs/terms/edw.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Lastly, the somewhat exponential adoption of cloud data warehouses in the last decade has shifted the terminology from what many people called an EDW to a data warehouse.
```
Comment (KiraFuruichi) [inline website/docs/terms/edw.md:30]:
```suggestion

Lastly, the somewhat exponential adoption of cloud data warehouses in the last decade has shifted the terminology from what many people called an EDW to a data warehouse.
```

File: website/docs/terms/edw.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+EDWs also help in building a 360-degree view of the company by combining different sources of information, such as customer feedback, financial records, product inventory, and marketing insights. All of this information can then be organized in data marts, schemas, and tables within one EDW that are eventually exposed to a BI tool.
+
+In addition, because all of an organization’s data is stored in one place, data teams can provide access to only those who need access to specific schemas and tables. Keeping these access patterns and changes in only one data warehouse will limit the amount of data needed to go through for auditing and other security regulations.
+
+## Conclusion
+
+An enterprise data warehouse is, in general, like any other data warehouse; it acts as a central home for multiple departments’ raw and transformed data. An EDW is often composed of multiple databases to store raw, staging, development, and production-ready data. The primary benefits for an EDW are centralization, standardization, and accessibility. You probably have a data warehouse setup like an EDW, you’re likely just not calling it that 😉
+
+
+## Additional reading
```
Comment (KiraFuruichi) [inline website/docs/terms/edw.md:54]:
```suggestion
## Additional reading
EDW, data warehouse, or something different altogether? Check out some of our favorite resources on the fundamental of data storage and organization:
```

**Review by KiraFuruichi — CHANGES_REQUESTED** [review]
Minor edits to match draft

**Comment by johnblust** [comment]
Should be good to go now!

**Review by KiraFuruichi — APPROVED** [review]
Amazing, thanks! Think `current` had some updates, so feel free to merge in once this branch is updated!

**Pull request merged**